index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+    <head>
```

Comment:
Kindly add some of the missing elements in the head section, for example, 'charset' attribute.